src/main/java/duckie/exception/DuckieFileErrorException.java

```diff
@@ -0,0 +1,15 @@
+package duckie.exception;
+
+/**
+ * DuckieException thrown when facing errors loading duckie file
+ */
+public class DuckieFileErrorException extends DuckieException {
+    protected static final String INDENT = "\t";
```

Comment:
This INDENT variable could be initialized just once in the DuckieException and the used in all the other exceptions that extend it.